pkg/kubelet/volumemanager/volume_manager.go

```diff
@@ -421,18 +421,21 @@ func (vm *volumeManager) WaitForAttachAndMount(pod *v1.Pod) error {
 	if err != nil {
 		unmountedVolumes :=
 			vm.getUnmountedVolumes(uniquePodName, expectedVolumes)
-		// Also get unattached volumes for error message
+		// Also get unattached volumes and volumes not in dsw for error message
 		unattachedVolumes :=
-			vm.getUnattachedVolumes(expectedVolumes)
+			vm.getUnattachedVolumes(uniquePodName)
+		volumesNotInDSW :=
+			vm.getVolumesNotInDSW(uniquePodName, expectedVolumes)
```

Comment:
Why we needs ``VolumesNotInDSW`` check?

Reply:
Should we be sure that an unsuccessful volume process event is only generated in the ``attach`` and ``mount``stages in Kubelet?

Reply:
> Why we needs VolumesNotInDSW check?
Because sometimes DSW of the volume is not generated successfully, the ut covered this scenario, you could take a look. I think it is useful for troubleshooting.